packages/calypso-products/src/is-plan.ts

```diff
@@ -0,0 +1,12 @@
+import { PLAN_HOST_BUNDLE, PLAN_FREE } from './constants';
+import { getPlansSlugs } from './main';
+
+type SnakeCaseProduct = { product_slug: string };
+type CamelCaseProduct = { productSlug: string };
+export function isPlan( product: SnakeCaseProduct | CamelCaseProduct ): boolean {
+	const slug = 'product_slug' in product ? product.product_slug : product.productSlug;
+	if ( slug === PLAN_FREE ) {
+		return false;
+	}
+	return getPlansSlugs().includes( slug ) || slug === PLAN_HOST_BUNDLE;
```

Comment:
If I understand correctly I don't think this will pick up the enterprise plan. I'm not sure what that is, but it would change the function's behavior.

What do you think about using a `switch` here, something like this:

```
switch ( slug ) {
    case PLAN_FREE:
        return false;
    case PLAN_HOST_BUNDLE:
    case PLAN_WPCOM_ENTERPRISE:
        return true;
    default:
        return getPlansSlugs().includes( slug );
}
```

Reply:
Wow, you're right. The enterprise plan was an old plan that's been removed. I didn't notice that it wasn't in `PLANS_LIST`.